src/ducks/Signals/utils/utils.js

```diff
@@ -388,14 +387,12 @@ const getFrequencyFromCooldown = ({ cooldown }) => {
   }
 }
 
-export const mapTriggerTarget = (
-  target,
-  { value } = {},
-  address = '',
-  isEthWalletTrigger = false
-) => {
-  let newTarget
+export const getTargetFromArray = target =>
+  target.length === 1 ? target[0].slug : target.map(({ slug }) => slug)
 
+export const mapTriggerTarget = (target, signalType = {}, address) => {
+  const { value } = signalType
+  let newTarget
```

Comment:
Why do use switch here with one option. And why do you need to have temp var. You can return value instead of break. Or just use simple if, with more readability than switch, default, new var. @GarageInc